src/CMakeLists.txt

```diff
@@ -1,8 +1,19 @@
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(EventdEvent REQUIRED libeventd-event>=0.3)
+pkg_check_modules(EventdEvent REQUIRED libeventd>=0.3)
 pkg_check_modules(EventdPlugin REQUIRED libeventd-plugin>=0.3)
 pkg_check_modules(SystemdJournal REQUIRED libsystemd-journal)
 
+macro(pkg_config_variable PKG VAR OUT)
```

Comment:
[Patch made](https://github.com/mathstuf/CMake/commit/70e8db6e20749a484dd677d7094780c5f4b451c6); will shepard it tomorrow.